### PR TITLE
Added ament_lint_common

### PR DESCRIPTION
**Pull request by ahcorde**
Added `ament_lint_common`

Signed-off-by: ahcorde <ahcorde@gmail.com>

### CODE REVIEW

**Comment by ahcorde** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/10311/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/5842/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/8398/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/10218/)

File: CMakeLists.txt [hunk 1]
```diff
@@ -16,6 +16,7 @@ find_package(rosidl_default_generators REQUIRED)
 rosidl_generate_interfaces(${PROJECT_NAME}
   "action/Fibonacci.action"
   "srv/AddTwoInts.srv"
+  ADD_LINTER_TESTS
```
Comment (dirk-thomas) [inline CMakeLists.txt:19]: Why should the example interfaces be linted? I don't think we want to recommend this usage to users.

Comment (ahcorde) [reply CMakeLists.txt:19]: `example_interfaces` is being used in some packages (for testing)  who aspire to be a Quality Level 1 packages. That's why I decided to add the linters.

Feel free to close it if you consider that this change is not adding any value.

Comment (dirk-thomas) [reply CMakeLists.txt:19]: I don't see how the linters are relevant for any quality level?

Comment (ahcorde) [reply CMakeLists.txt:19]: Linters are defined as part of the quality level of a package https://github.com/ros-infrastructure/rep/blob/e421be8db3ef6fa65b29f9325ce635afdaecbf16/rep-2004.rst#iii-a

without linters, a package could not be more than quality 3, does make sense to test code with Level 1 or Level 2 with poor quality level package?

Comment (dirk-thomas) [reply CMakeLists.txt:19]: I am pretty sure the REP assumes linters for code the package actually writes. Maybe that needs to be clarified in the REP.

**Pull request closed without merging**